docs/terraformoptions.adoc

```diff
@@ -739,14 +739,9 @@ Refer to {uri-topology}[topology] for more thorough examples.
 node_pools = {
 np1 = { shape = "VM.Standard.E4.Flex", ocpus = 1, memory = 16, node_pool_size = 1, boot_volume_size = 150, label = { app = "frontend", pool = "np1" } }
   np2 = {shape="VM.Standard.E2.2",node_pool_size=2,boot_volume_size=150,label={app="application",name="test"}}
-  np3 = {shape="VM.Standard.E2.2",node_pool_size=1}
-}
-|
-node_pools = {
-  np1 = {shape="VM.Standard.E3.Flex",ocpus=2,node_pool_size=2,boot_volume_size=150}
-  np2 = {shape="VM.Standard.E2.2",node_pool_size=2,boot_volume_size=150,label={app="application",name="test"}}
-  np3 = {shape="VM.Standard.E2.2",node_pool_size=1}
-}
+  np3 = {shape="VM.Standard.E2.2",node_pool_size=1,placement_ADs=[1]} 
```

Comment:
Please use snake_case like the other parameters

Reply:
This is snake_case in my mind, would you prefer `placement_ads`?

Reply:
yes please